docs/src/components/Playground/Playground.js

```diff
@@ -468,6 +473,78 @@ export default class Playground extends Component {
           </CardGroup>
         </PageBlock>
 
+        <PageBlock>
+          <Section header>
+            <Text hero>Icons</Text>
```

Comment:
Love that you added this to the playground—makes reviewing this so much easier 👏👏👏